src/plugins/implicitRelationships.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { findByProps, findStoreLazy } from "@webpack";
+import { ChannelStore, FluxDispatcher, GuildStore, RelationshipStore, UserStore } from "@webpack/common";
+
+const UserAffinitiesStore = findStoreLazy("UserAffinitiesStore");
+
+export default definePlugin({
+    name: "ImplicitRelationships",
+    description: "Shows your implicit relationships in the Friends tab.",
+    authors: [Devs.Dolfies],
+    patches: [
+        {
+            find: "FriendsEmptyState: Invalid empty state",
+            replacement: [
+                // Counts header
+                {
+                    match: /toString\(\)\}\);case (\i\.\i)\.BLOCKED/,
+                    replace: "toString()});case $1.IMPLICIT:return \"Implicit — \"+arguments[1].toString();case $1.BLOCKED"
+                },
+                // No friends page
+                {
+                    match: /case (\i\.\i)\.ONLINE:return (\i)\.SECTION_ONLINE/,
+                    replace: "case $1.ONLINE:case $1.IMPLICIT:return $2.SECTION_ONLINE"
+                },
+                // Sections header
+                {
+                    match: /\(0,(\i)\.jsx\)\((\i)\.TabBar\.Item,\{id:(\i)\.(\i)\.BLOCKED,className:([^\s]+?)\.item,children:\i\.\i\.Messages\.BLOCKED\}\)/,
+                    replace: "(0,$1.jsx)($2.TabBar.Item,{id:$3.$4.IMPLICIT,className:$5.item,children:\"Implicit\"}),$&"
+                },
+                // Sections content
+                {
+                    match: /case (\i\.\i)\.BLOCKED:return (\i)\.type===(\i\.\i)\.BLOCKED/,
+                    replace: "case $1.BLOCKED:return $2.type===$3.BLOCKED;case $1.IMPLICIT:return $2.type===5"
```

Comment:
now you don't need any of their jazz
```suggestion
                    replace: ";case $1.IMPLICIT:return $2.type===5"
```